dev/utils.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
```

Comment:
What about moving this to under `dev/sparktestsupport`? That directory has some util-ish modules related to tests and CI.

Reply:
Sure!